models/User.js

```diff
@@ -4,6 +4,7 @@ const { Schema, model } = mongoose;
 const UserSchema = new Schema(
   {
     username: { type: String, required: true },
+    aboutMe: String,
```

Comment:
we did not set a required false and so we need to make it unique?

Reply:
I think required false is the default and why unique?

Reply:
ah ok. well that you cant copy the description of someone else? its picky but..